index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="pt_BR">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Home | Apeperia</title>
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Montserrat:300,400,700|Open+Sans:300,400,700&display=swap">
+    <link rel="stylesheet" href="css/reset.css">
+    <link rel="stylesheet" href="css/base.css">
+</head>
+<body>
+    <header>
+        <a href="/index.html">
+            <img src="img/logo-apeperia.svg" alt="Logo da Apeperia">

```

Comment:
Ficamos com Apeperia mesmo? Achei que fosse ficar com o outro que o bugan tinha passado por já ter o layout pra mobile (não tem problema se for ficar nesse, mais pra confirmar hehe)